services/director/tests/test_producer.py

```diff
@@ -61,6 +61,7 @@ async def push_start_services(number_comp: int, number_dyn: int, dependant=False
                 service_version,
                 service_uuid,
                 service_basepath,
+                "",
```

Comment:
mmhhh this means Not set? I rather use `None` to do that! Otherwise we end up with the ``thumbnail`` problem

Reply:
This is what we already had since this was introduced. It's either `puppeteer` or empty string. I'm just propagating it currently.